From 6918d2e111ca36402843572021a66a9998133fc3 Mon Sep 17 00:00:00 2001
From: Alexey Chernov <4ernov@gmail.com>
Date: Sat, 4 Jul 2020 01:02:39 +0300
Subject: [PATCH] FHS-friendly LV2 and VST3 plugin installation paths

Make plugin installation paths compatible with FHS on certain
distros.
---
 plugin/lv2/CMakeLists.txt | 4 ++--
 plugin/vst/CMakeLists.txt | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/plugin/lv2/CMakeLists.txt b/plugin/lv2/CMakeLists.txt
index bcde781..e331381 100644
--- a/plugin/lv2/CMakeLists.txt
+++ b/plugin/lv2/CMakeLists.txt
@@ -10,5 +10,5 @@ target_link_libraries(geonkick_lv2 "-lstdc++fs")
 target_link_libraries(geonkick_lv2 "-lredkite -lX11 -lsndfile -lrt -lm -ldl -lpthread -lcairo")
 add_dependencies(geonkick_lv2 api_plugin)
 
-install(DIRECTORY ${GKICK_LV2_DIR}/geonkick.lv2 DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/lv2)
-install(TARGETS geonkick_lv2 DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/lv2/geonkick.lv2)
+install(DIRECTORY ${GKICK_LV2_DIR}/geonkick.lv2 DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/lv2)
+install(TARGETS geonkick_lv2 DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/lv2/geonkick.lv2)
diff --git a/plugin/vst/CMakeLists.txt b/plugin/vst/CMakeLists.txt
index ecfe679..a170030 100644
--- a/plugin/vst/CMakeLists.txt
+++ b/plugin/vst/CMakeLists.txt
@@ -40,4 +40,4 @@ target_link_libraries(${target} PRIVATE "-Wl,--no-undefined")
 
 add_dependencies(${target} geonkick_common)
 
-install(TARGETS geonkick_vst DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/vst3/geonkick.vst3/Contents/${LINUX_ARCHITECTURE_NAME})
+install(TARGETS geonkick_vst DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/vst3/geonkick.vst3/Contents/${LINUX_ARCHITECTURE_NAME})
